torch/ao/quantization/fx/backend_config/native.py

```diff
@@ -303,6 +304,22 @@ def _get_conv_configs():
     ],
 }
 
+_HARDSIGMOID_MODULE_CONFIG = {
+    "pattern": torch.nn.Hardsigmoid,
+    "observation_type": ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT,
+    # TODO: The following two keys are temporary, since we don't want to put observer in the configs
+    # we expect that it's provided by user
+    # What we want to put here is the requirement on observers, in this case dtype,
+    # quant_min, quant_max etc., but we need to first move all configs to
```

Comment:
quant_min, quant_max are internal variables to observers.  I think restrictions in backend_config_dict should be on the public API only, such as scale and zero_point.

Reply:
OK, makes sense, yeah we can put scale/zero_point constraints here in the future